### DISCUSSION TITLE
How to extend the site search to all pages other than just the posts?

### DISCUSSION THREAD

**Discussion by EverettYou** [sylhare/Type-on-Strap#408] [Q&A] [answered]
The site search currently only searches in the posts. Can it be extended to all pages?

**Comment by sylhare** [comment]
Yes I believe it could. The search is based on a json file [search.json](https://github.com/sylhare/Type-on-Strap/blob/f531e82eca47f594eb1e55c707cf2a6aeadb5ef8/assets/data/search.liquid) that is created via liquid markup.
It creates it from the posts and the portfolio collection. If you follow the same logic you could also loop through the `site.pages` and add that to the json.

**Comment by sylhare** [comment, marked as answer]
@EverettYou I made a PR to add the page to the search: https://github.com/sylhare/Type-on-Strap/pull/418 so it will be available by default once merged.
